### PR TITLE
Feat/javascript rest api return top hits

### PR DESCRIPTION
**Pull request by Doysch**
Added functionality to determine the Analytics API region through an environmental variable depending on geographical area - 2 options 'de' or 'us'

**Contributing to Algolia Samples**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: javascript/rest_API_return_top_1000_hits.js [hunk 1]
```diff
@@ -4,34 +4,39 @@ This script makes a GET request to the /searches endpoint on the Analytics REST
 To get the top 1000 searches over the last 7 days.
 There is no API client for Analytics, so this script uses the JavaScript Requests library to make the call.
 */
-
+try {
 // Install the API client: https://www.algolia.com/doc/api-client/getting-started/install/javascript/?client=javascript
 const algoliasearch = require("algoliasearch");
```
Comment (christopherjstock) [inline javascript/rest_API_return_top_1000_hits.js:9]: This import isn't needed. There's no api client for the analytics endpoint, we'll make direct calls to the REST api.

File: javascript/rest_API_return_top_1000_hits.js [hunk 2]
```diff
@@ -4,34 +4,39 @@ This script makes a GET request to the /searches endpoint on the Analytics REST
...
 // Get your Algolia Application ID and (admin) API key from the dashboard: https://www.algolia.com/account/api-keys
-// and choose a name for your index. Add these environment variables to a `.env` file:
+// use the name of the index you want to target. Add these environment variables to the `.env` file:
 const ALGOLIA_APP_ID = process.env.ALGOLIA_APP_ID;
 const ALGOLIA_API_KEY = process.env.ALGOLIA_API_KEY;
 const ALGOLIA_INDEX_NAME = process.env.ALGOLIA_INDEX_NAME;
+const URL_DOMAIN = process.env.URL_DOMAIN;
+
 const fs = require("fs");
 const { fileURLToPath } = require("url");
```
Comment (christopherjstock) [inline javascript/rest_API_return_top_1000_hits.js:21]: This variable isn't used.

**Review by christopherjstock — COMMENTED** [review]
Script works great, just some minor housekeeping needed.

File: javascript/rest_API_return_top_1000_hits.js (hunk 1 above)
Comment (Doysch) [reply javascript/rest_API_return_top_1000_hits.js:9]: Done @christopherjstock  - thanks!

**Review by matthewbond — APPROVED** [review]

**Pull request merged**